plugins/retry/src/index.ts

```diff
@@ -113,6 +113,7 @@ export function PiniaColadaRetry(
           if (shouldRetry) {
             const delayTime = typeof delay === 'function' ? delay(entry.retryCount) : delay
             entry.timeoutId = setTimeout(() => {
+              if (!queryEntry.active) return
```

Comment:
shouldn't it also delete from the retryMap in this case to reset the count?
It would be great to add a failing test case first where the query with the same key is mounted again and see that the retry behavior is reset 🤔

Reply:
Thanks for pointing that out! It makes sense to delete it from the retryMap to reset the count.